tests/console/shells.pm

```diff
@@ -14,12 +14,16 @@ use base "consoletest";
 use strict;
 use testapi;
 use utils 'zypper_call';
+use version_utils 'is_leap';
 
 sub run() {
     select_console 'root-console';
-    zypper_call 'in ksh tcsh zsh';
+    my @packages = qw(tcsh zsh);
+    # ksh does not build for Leap 15.x on aarch64, so, skip it
+    push @packages, qw(ksh) unless (is_leap('15.0+') and get_var('ARCH') =~ /aarch64/);
```

Comment:
IMHO, it could be better to use `check_var('ARCH', 'aarch64')` instead of `get_var`.

Not sure that `qw` is needed to only add ksh, but that should not hurt :)